libraries/botbuilder-dialogs/tests/activityPrompt.test.js

```diff
@@ -162,7 +162,7 @@ describe('ActivityPrompt', function () {
 
             const results = await dc.continueDialog();
             if (results.status === DialogTurnStatus.empty) {
-                await dc.prompt('prompt', 'Please send an activity.');
+                await dc.prompt('prompt', { prompt: 'Please send an activity.', retryPrompt: 'Activity not received.' });
```

Comment:
Can you update the tests to match @EricDahlvang's changes in .NET?

https://github.com/microsoft/botbuilder-dotnet/pull/4298/files#diff-2f1e96fad160f0d167a84f3c85ba687c

Reply:
Done